### PR TITLE
Update to scala 2.12

### PR DESCRIPTION
**Pull request by geoffjentry**
Passes a cromwell centaur run: https://travis-ci.org/broadinstitute/cromwell/builds/214892572

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,15 +16,15 @@ val akkaV = "2.4.11" // Note: akka-http branches from akkaV after 2.4.11
 // logBuffered in Test := false
 
 libraryDependencies ++= Seq(
-  "com.github.kxbmap" %% "configs" % "0.4.2",
+  "com.github.kxbmap" %% "configs" % "0.4.4",
   "com.typesafe" % "config" % "1.3.0",
-  "org.typelevel" %% "cats" % "0.7.2"
+  "org.typelevel" %% "cats" % "0.9.0"
```
Comment (mcovarr) [inline build.sbt:21]: 😸

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/170/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: build.sbt (hunk 1 above)
Comment (geoffjentry) [reply build.sbt:21]: I was a little worried it'd not be at all drop in but the worst that happened was the removal of the default tailrec so hooray

Comment (mcovarr) [reply build.sbt:21]: Yeah I knew that was coming and am also pleasantly surprised that's the only substantive change required, at least in Centaur-land.

**Comment by kshakir** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/170/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**